packages/suite-native/src/screens/Home.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { Platform, SafeAreaView, ScrollView, StatusBar, useColorScheme, View } from 'react-native';
+
+import { Text, Box, Button } from '@trezor/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import TrezorConnect from 'trezor-connect';
+
+const backgroundStyle = prepareNativeStyle<{ isDarkMode: boolean }>(
+    ({ colors, spacings }, { isDarkMode }) => ({
+        backgroundColor: isDarkMode ? colors.black : colors.white,
+        padding: spacings.lg,
+        marginTop: 0,
+    }),
+);
+
+const connectOptions = {
+    transportReconnect: true,
+    debug: true,
+    manifest: {
+        email: 'info@trezor.io',
+        appUrl: '@trezor/suite',
+    },
+};
+
+export const Home = () => {
+    const isDarkMode = useColorScheme() === 'dark';
+    const [connectStatus, setConnectStatus] = useState<any>('');
+    const [deviceState, setDeviceState] = useState<any>('');
```

Comment:
Usage and default value indicate that these state properties should be type `string` instead of `any`. Maybe you forgot to fix these typings?

Reply:
Would be better, but this page is just some "dev" showcase and this page will be probably deleted soon